minos/cli/templating/processors.py

```diff
@@ -136,6 +138,7 @@ def answers(self) -> dict[str, Any]:
         """
         answers = self.context
         answers |= self._previous_answers
+        answers |= {"template_registry": f"{TEMPLATE_URL}/{TEMPLATE_VERSION}", "template_version": TEMPLATE_VERSION}
```

Comment:
Hi @albamig, instead of overriding those variables, you can pop them from previous answers, which is the reason why are modified from the ones provided by the `TemplateFetcher.metadata`

You can do that with `self._previous_answers.pop("template_registry", None)` and `self._previous_answers.pop("template_version", None)`. 

The best place to do that is directly in the `_previous_answers` method.

Reply:
Okay, I agree.

I did it there so it's very visible that we're not using a cached `template_registry`. But popping instead of overriding is a cleaner solution.